### PR TITLE
Move to GridPane wallet, dynamic sizing

### PR DESCRIPTION
**Pull request by user411**
Wallet layout changes and some dynamic sizing to support more window sizes

Screen Shot 2021-08-05 at 2 18 22 PM

### CODE REVIEW

File: app/bundle/src/main/scala/org/bitcoins/bundle/gui/BundleGUI.scala [hunk 1]
```diff
@@ -34,7 +34,6 @@ object BundleGUI extends WalletGUI with BitcoinSAppJFX3 {
 
   val DEFAULT_WIDTH = 1400.0
   val DEFAULT_HEIGHT = 800.0
-  val DEFAULT_MAXIMIZED = false
```
Comment (user411) [inline app/bundle/src/main/scala/org/bitcoins/bundle/gui/BundleGUI.scala:37]: Code cleanup, this preference didn't end up getting persisted

File: core/src/main/scala/org/bitcoins/core/dlc/accounting/RateOfReturnUtil.scala [hunk 2]
```diff
@@ -7,4 +7,10 @@ object RateOfReturnUtil {
     val percent = ror * 100
     f"${percent}%1.2f" + "%"
   }
+
```
Comment (benthecarman) [inline core/src/main/scala/org/bitcoins/core/dlc/accounting/RateOfReturnUtil.scala:9]: can make the above function now:

```scala
def prettyPrint(ror: BigDecimal): String = {
    toPercentage(ror, 2)
  }
```

**Review by benthecarman — COMMENTED** [review]
otherwise lgtm

File: core/src/main/scala/org/bitcoins/core/dlc/accounting/RateOfReturnUtil.scala (hunk 2 above)
Comment (user411) [reply core/src/main/scala/org/bitcoins/core/dlc/accounting/RateOfReturnUtil.scala:9]: Added commit to reuse toPercentage()

**Review by benthecarman — APPROVED** [review]

**Pull request merged**